ci: DLT-1602 release notification

juliodialpad opened the pull request:
# Release notification update

## Obligatory GIF (super important!)

![Obligatory GIF](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaGFlYmMwZDByMTRwejhxYXA1MWZ4MXh5ZWVjb25jcjAyY3RoYjA1biZlcD12MV9naWZzX3RyZW5kaW5nJmN0PWc/2IKLaqioz2ZRAngxFA/giphy.gif)

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] CI

## :book: Jira Ticket

- https://dialpad.atlassian.net/browse/DLT-1602

## :book: Description

- Updated release notification to send the message directly to `#dialtone` channel
- Removed unneeded filter and `fetch-depth` from `release.yml`
- Added API_URL env variable to `send-blog-communications.yml`
- Updated `release-ci.config.js` files on emojis and icons packages to detect changes on ci too.
- Linter changes on `gulpfile.cjs`
- Updated the release script to fix issues.

Note: All the commits from alpha/beta were merged back due to a mistake on the release script, removed the merge command from the release script to avoid having those commits on staging on later alpha/beta releases.

## :bulb: Context

- Release notificatio re-work after testing on the test channel.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

ryandialpad commented:
@dialpad-drew Adding you as a second pair of 👀  to ensure that we are not accidentally merging in stale code

ryandialpad commented:
Changes to

```bash
modal.json
modal.html
modal.less
```

look good

juliodialpad committed:
fetch all history

juliodialpad committed:
chore: NO-JIRA avoid building on local and github release

juliodialpad committed:
ci: fetch all history

juliodialpad committed:
don't build on local release

juliodialpad committed:
change base branch base on release type

juliodialpad committed:
push clean alpha branch to make semantic-release work

juliodialpad committed:
stop running the release if no relevant changes

juliodialpad committed:
update release script

juliodialpad committed:
do only dialtone to test faster

juliodialpad committed:
remove dry-run

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
fix not detecting changes

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
fix: test change

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Bug Fixes

* test change ([ffb5420](https://github.com/dialpad/dialtone/commit/ffb5420290db00190fb829486a071f5a99c69be1))

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
add changelog to change detection filter

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Bug Fixes

* test change ([ffb5420](https://github.com/dialpad/dialtone/commit/ffb5420290db00190fb829486a071f5a99c69be1))

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
fix filters

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Bug Fixes

* test change ([ffb5420](https://github.com/dialpad/dialtone/commit/ffb5420290db00190fb829486a071f5a99c69be1))

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
change if logic

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.1

# [9.34.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.33.0...dialtone/v9.34.0-beta.1) (2024-05-02)

### Bug Fixes

* test change ([ffb5420](https://github.com/dialpad/dialtone/commit/ffb5420290db00190fb829486a071f5a99c69be1))

### Features

* DLT-1682 add channel-api icon ([#279](https://github.com/dialpad/dialtone/issues/279)) ([d909531](https://github.com/dialpad/dialtone/commit/d9095311e878dc03b19f5f02375790b6bd038743))

juliodialpad committed:
fix: remove test change

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.0-beta.2

# [9.34.0-beta.2](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.0-beta.1...dialtone/v9.34.0-beta.2) (2024-05-02)

### Bug Fixes

* remove test change ([d7a45c2](https://github.com/dialpad/dialtone/commit/d7a45c2b70c39048f8e391cd38f859d200ee4c9f))

juliodialpad committed:
refactor: linter changes

juliodialpad committed:
update rules on release-ci

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.15.1-alpha.1

## [4.15.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.15.0...dialtone-icons/v4.15.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.1

## [9.34.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.0...dialtone/v9.34.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

juliodialpad committed:
fix release version parsing

juliodialpad committed:
refactor: fix release action

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.15.1-alpha.1

## [4.15.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.15.0...dialtone-icons/v4.15.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.2

## [9.34.1-alpha.2](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.1...dialtone/v9.34.1-alpha.2) (2024-05-03)

### Code Refactoring

* fix release action ([eb733b5](https://github.com/dialpad/dialtone/commit/eb733b5f43cb46fbedaea28cb5e045d451fd8f7d))

juliodialpad committed:
refactor: change release message

juliodialpad committed:
update release script

juliodialpad committed:
update release script

(cherry picked from commit 320f8ec38b45ad9b78cb3d350f89c8d8757bf45a)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.15.1-alpha.1

## [4.15.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.15.0...dialtone-icons/v4.15.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.3

## [9.34.1-alpha.3](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.2...dialtone/v9.34.1-alpha.3) (2024-05-03)

### Code Refactoring

* change release message ([f128619](https://github.com/dialpad/dialtone/commit/f128619fa3e9940d804f5ef89938eccf734a0055))

juliodialpad committed:
refactor: fix indentation

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.15.1-alpha.1

## [4.15.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.15.0...dialtone-icons/v4.15.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.4

## [9.34.1-alpha.4](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.3...dialtone/v9.34.1-alpha.4) (2024-05-03)

### Code Refactoring

* fix indentation ([f6734d3](https://github.com/dialpad/dialtone/commit/f6734d3b3747f1b915d46d059ff7e3b06499c52e))

juliodialpad committed:
refactor: add quotes

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.15.1-alpha.1

## [4.15.1-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.15.0...dialtone-icons/v4.15.1-alpha.1) (2024-05-03)

### Code Refactoring

* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.5

## [9.34.1-alpha.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.4...dialtone/v9.34.1-alpha.5) (2024-05-03)

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))

juliodialpad committed:
refactor: scape quotes

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.5

## [9.34.1-alpha.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.4...dialtone/v9.34.1-alpha.5) (2024-05-03)

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))
* scape quotes ([786f3cd](https://github.com/dialpad/dialtone/commit/786f3cd1829b75be83210fe2ce1bc61d2e2d9f5f))

juliodialpad committed:
refactor: test release

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.5

## [9.34.1-alpha.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.4...dialtone/v9.34.1-alpha.5) (2024-05-03)

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))
* scape quotes ([786f3cd](https://github.com/dialpad/dialtone/commit/786f3cd1829b75be83210fe2ce1bc61d2e2d9f5f))
* test release ([971bd9b](https://github.com/dialpad/dialtone/commit/971bd9bc40cc05e39b3c4ef14213cbef647e8992))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.34.1-alpha.5

## [9.34.1-alpha.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.4...dialtone/v9.34.1-alpha.5) (2024-05-03)

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))
* scape quotes ([786f3cd](https://github.com/dialpad/dialtone/commit/786f3cd1829b75be83210fe2ce1bc61d2e2d9f5f))
* test release ([971bd9b](https://github.com/dialpad/dialtone/commit/971bd9bc40cc05e39b3c4ef14213cbef647e8992))

juliodialpad committed:
feat: test release

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.1

# [9.35.0-alpha.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1-alpha.4...dialtone/v9.35.0-alpha.1) (2024-05-03)

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))
* scape quotes ([786f3cd](https://github.com/dialpad/dialtone/commit/786f3cd1829b75be83210fe2ce1bc61d2e2d9f5f))
* test release ([971bd9b](https://github.com/dialpad/dialtone/commit/971bd9bc40cc05e39b3c4ef14213cbef647e8992))

### Features

* test release ([7a07314](https://github.com/dialpad/dialtone/commit/7a07314ec402a18b0cbc603f516c9245d0753112))

juliodialpad committed:
fix: test HTML

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.2

# [9.35.0-alpha.2](https://github.com/dialpad/dialtone/compare/dialtone/v9.35.0-alpha.1...dialtone/v9.35.0-alpha.2) (2024-05-06)

### Bug Fixes

* test HTML ([61e2579](https://github.com/dialpad/dialtone/commit/61e2579441aeb5ea396a0a302e319cfa59f36643))

juliodialpad committed:
Merge branch 'refs/heads/staging' into test-notification-release-action

# Conflicts:
#	scripts/release.sh

juliodialpad committed:
fix: NO-JIRA test sanitized release notes

juliodialpad committed:
ci: fix alpha/beta release

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.3

# [9.35.0-alpha.3](https://github.com/dialpad/dialtone/compare/dialtone/v9.35.0-alpha.2...dialtone/v9.35.0-alpha.3) (2024-05-06)

### Bug Fixes

* **Message Input:** DLT-1711 focus message input on emoji select ([#297](https://github.com/dialpad/dialtone/issues/297)) ([724acd5](https://github.com/dialpad/dialtone/commit/724acd51e8b7e35eaf60f1ec493c4111dcb94e78))
* NO-JIRA test sanitized release notes ([ee6412f](https://github.com/dialpad/dialtone/commit/ee6412f311dd65d35da99cf967c6fb4fafd7c7ee))
* **Tooltip:** DLT-1747 tooltip staying open over overlay ([#299](https://github.com/dialpad/dialtone/issues/299)) ([3233c11](https://github.com/dialpad/dialtone/commit/3233c11a000b6088fc62095354d46f49c778aef9))

juliodialpad committed:
fix: try with code block

juliodialpad committed:
Merge branch 'refs/heads/alpha' into test-notification-release-action

juliodialpad committed:
ci: update release script

juliodialpad committed:
fix: remove parentheses

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.4

# [9.35.0-alpha.4](https://github.com/dialpad/dialtone/compare/dialtone/v9.35.0-alpha.3...dialtone/v9.35.0-alpha.4) (2024-05-06)

### Bug Fixes

* remove parentheses ([0f6807b](https://github.com/dialpad/dialtone/commit/0f6807b0528d23072ceabe2db2a04e397e185836))
* try with code block ([302b802](https://github.com/dialpad/dialtone/commit/302b8021b28d561a688cb1990d976de0fa5be4da))

juliodialpad committed:
fix: missing dollar sign

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.5

# [9.35.0-alpha.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.35.0-alpha.4...dialtone/v9.35.0-alpha.5) (2024-05-06)

### Bug Fixes

* missing dollar sign ([b4b5fd1](https://github.com/dialpad/dialtone/commit/b4b5fd1cda3eda27449ea899efb408e57ee42c5d))

juliodialpad committed:
fix: spacing

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-alpha.6

# [9.35.0-alpha.6](https://github.com/dialpad/dialtone/compare/dialtone/v9.35.0-alpha.5...dialtone/v9.35.0-alpha.6) (2024-05-06)

### Bug Fixes

* spacing ([3bf9f6c](https://github.com/dialpad/dialtone/commit/3bf9f6cc982fac664f69b13e004e68db28502eec))

juliodialpad committed:
update release notification

juliodialpad committed:
Merge branch 'ci/release-notification' into alpha

juliodialpad committed:
Merge branch 'ci/release-notification' into beta

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.35.0-beta.1

# [9.35.0-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.34.1...dialtone/v9.35.0-beta.1) (2024-05-06)

### Bug Fixes

* missing dollar sign ([b4b5fd1](https://github.com/dialpad/dialtone/commit/b4b5fd1cda3eda27449ea899efb408e57ee42c5d))
* NO-JIRA test sanitized release notes ([ee6412f](https://github.com/dialpad/dialtone/commit/ee6412f311dd65d35da99cf967c6fb4fafd7c7ee))
* remove parentheses ([0f6807b](https://github.com/dialpad/dialtone/commit/0f6807b0528d23072ceabe2db2a04e397e185836))
* remove test change ([d7a45c2](https://github.com/dialpad/dialtone/commit/d7a45c2b70c39048f8e391cd38f859d200ee4c9f))
* spacing ([3bf9f6c](https://github.com/dialpad/dialtone/commit/3bf9f6cc982fac664f69b13e004e68db28502eec))
* test change ([ffb5420](https://github.com/dialpad/dialtone/commit/ffb5420290db00190fb829486a071f5a99c69be1))
* test HTML ([61e2579](https://github.com/dialpad/dialtone/commit/61e2579441aeb5ea396a0a302e319cfa59f36643))
* try with code block ([302b802](https://github.com/dialpad/dialtone/commit/302b8021b28d561a688cb1990d976de0fa5be4da))

### Code Refactoring

* add quotes ([4125189](https://github.com/dialpad/dialtone/commit/4125189ae1ca313cc597b6353b1c38586ba46c7f))
* change release message ([f128619](https://github.com/dialpad/dialtone/commit/f128619fa3e9940d804f5ef89938eccf734a0055))
* fix indentation ([f6734d3](https://github.com/dialpad/dialtone/commit/f6734d3b3747f1b915d46d059ff7e3b06499c52e))
* fix release action ([eb733b5](https://github.com/dialpad/dialtone/commit/eb733b5f43cb46fbedaea28cb5e045d451fd8f7d))
* linter changes ([df290fe](https://github.com/dialpad/dialtone/commit/df290feaf001c8732ed17f8939348174dbe63f7b))
* scape quotes ([786f3cd](https://github.com/dialpad/dialtone/commit/786f3cd1829b75be83210fe2ce1bc61d2e2d9f5f))
* test release ([971bd9b](https://github.com/dialpad/dialtone/commit/971bd9bc40cc05e39b3c4ef14213cbef647e8992))

### Features

* test release ([7a07314](https://github.com/dialpad/dialtone/commit/7a07314ec402a18b0cbc603f516c9245d0753112))

juliodialpad committed:
remove unnecessary filter

juliodialpad committed:
remove merging alpha/beta commits back

braddialpad commented on CHANGELOG.md:
We need to get rid of this beta changelog? or just leave it?

juliodialpad replied:
We can leave it for now. Just pushed a change to avoid merging changes back from alpha/beta.

I can clean it up if needed tho.

braddialpad commented on packages/dialtone-icons/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dialpad/dialtone-icons",
-  "version": "4.15.0",
+  "version": "4.15.1-alpha.1",
```
alpha version set here

juliodialpad replied:
Shouldn't be an issue as with the next release it'd be overwritten. I can try cleaning it up if needed.

braddialpad reviewed (approved):
Looks good other than some of the weird beta release stuff still in here.

juliodialpad commented:
> Looks good other than some of the weird beta release stuff still in here.

Yeah, I was merging the changes back from alpha/beta branches trying to keep them up-to-date to avoid issues, but later found that I could use `merge -X theirs` to ignore conflicts when merging the feature branch with alpha/beta.

That way we're going to maintain alpha/beta releases on its corresponding branch.

It'd take some time to clean it up and I don't think it's necessary, semantic-release will correct this on the next release.

braddialpad commented:
:tada: This PR is included in version 9.35.0-beta.2 :tada:

The release is available on [GitHub release](https://github.com/dialpad/dialtone/releases/tag/dialtone/v9.35.0-beta.2)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
